app/views/layouts/_google_analytics.html.erb

```diff
@@ -1,12 +1,11 @@
 <% if ENV['GOOGLE_ANALYTICS_ID'].present? %>
+<!-- Global Site Tag (gtag.js) - Google Analytics -->
+<script async src='<%= "https://www.googletagmanager.com/gtag/js?id=#{ENV['GOOGLE_ANALYTICS_ID']}" %>'></script>
 <script>
-    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-            (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments)};
+  gtag('js', new Date());
 
-    ga('create', '<%= ENV['GOOGLE_ANALYTICS_ID'] %>', 'auto');
-    ga('send', 'pageview');
-    ga('require', 'ecommerce');
+  gtag('config', "<%= ENV['GOOGLE_ANALYTICS_ID'] %>");
```

Comment:
Totally unimportant, but these are the only double quotes in this script, everything else uses single quotes.